src/components/TopBar/components/Search/Search.scss

```diff
@@ -39,7 +40,6 @@
 
 .Results {
   position: relative;
-  z-index: z-index(nav, $fixed-element-stacking-order);
```

Comment:
This move might warrant a changelog addition though

Reply:
It's still there though, it's just moved to a parent-parent element. It shouuuldn't affect any consumers of the component.